verifyFunctionWarning.m

```diff
@@ -6,15 +6,15 @@
 %    success = verifyFunctionWarning('functionCall', warningMessage, varargin)
 %
 % INPUTS:
-%    functionCall:       The name of a function.
-%    warningMessage:     The warning message that we want to capture
+%    functionCall:       The name of a function
+%    warningMessage:     The specific warning message that need to be tested
 %
 % OUPUTS: 
-%    success:            Catch the provided warning message in tline
+%    success:            Verify if the function throws the provided warning message and catch it in tline
 %
 % EXAMPLE:
 %    % Test whether PaCER function throws the provided warning message: 
-%    verifyFunctionWarning('PaCER', warningMessage, 'inputs', {niiCT})
+%    assert(verifyFunctionWarning('PaCER', warningMessage, 'inputs', {niiCT}));
```

Comment:
This does not seem to be the right example for the usage of this function. Why use `assert`?